### DISCUSSION TITLE
How to apply my org's theme or logo for the workflow

### DISCUSSION THREAD

**Discussion by guru-web3** [Web3Auth/Web3Auth#316] [Help]
How to apply my org's theme or logo for the workflow

**Comment by ieow** [comment]
Web3Auth support whitelabeling which gives you the ability to have a consistent user experience throughout your application. Web3Auth allows you to define a custom UI, branding, and translations for your applications. All our SDKs support Whitelabeling, giving granular customisation capability across all our offerings.

You can do whitelabelling across Web3Auth Plug and Play Modal, Openlogin Adapter, and Torus Wallet Adapter instances.

[Web3Auth Plug and Play Modal](https://web3auth.io/docs/whitelabel/login-modal)
[Openlogin Adapter](https://web3auth.io/docs/whitelabel/user-flow-screens)
[Torus Wallet Adapter](https://web3auth.io/docs/whitelabel/torus-wallet-plugin)
